wp-content/themes/haruoshi_origin/functions.php

```diff
@@ -57,6 +57,14 @@ function enqueue_style() {
 			filemtime( get_stylesheet_directory() . '/dist/css/archive-works.css' ),
 		);
 	endif;
+	if ( get_post_type() === 'works' ) :
```

Comment:
この指定だとworks一覧ページでもsingle-works.cssが読み込まれてしまうので詳細ページでのみ読み込まれるようにしましょう
```suggestion
	if ( is_singular('works') ) :
```

Reply:
done https://github.com/oooshima/origin_wordpress/pull/28/commits/aba196be18770be61a69cd96b3a7335d3e981090